src/lib/ndpi_main.c

```diff
@@ -7422,6 +7424,7 @@ u_int16_t ndpi_guess_host_protocol_id(struct ndpi_detection_module_struct *ndpi_
 				      struct ndpi_flow_struct *flow) {
   struct ndpi_packet_struct *packet = &ndpi_str->packet;
   u_int16_t ret = NDPI_PROTOCOL_UNKNOWN;
+  int use_client = ndpi_str->cfg.use_client_ip_in_guess;
```

Comment:
... and here. Why using `ndpi_str->cfg.direction_detect_enabled` before and not now?
I think we can remove it; what do you think?

Reply:
Yeah I wasn't sure what would happen if direction detection was disabled? I thought the src_port wouldn't be matched to the client port.

Reply:
I'm not convinced, but let keep it this way for the time being